protos/mocap/mocap.proto

```diff
@@ -0,0 +1,131 @@
+      MAV_FRAME_LOCAL_OFFSET_NED = 7; // Offset to the current local frame. Anything expressed in this frame should be added to the current local frame position.
+      MAV_FRAME_GLOBAL_TERRAIN_ALT = 10; // Global (WGS84) coordinate frame with AGL altitude (at the waypoint coordinate).
+      MAV_FRAME_GLOBAL_TERRAIN_ALT_INT = 11; // Global (WGS84) coordinate frame (scaled) with AGL altitude (at the waypoint coordinate).
+      MAV_FRAME_BODY_FRD = 12; // Body fixed frame of reference, Z-down (x: forward, y: right, z: down). Replacement for MAV_FRAME_BODY_NED, MAV_FRAME_BODY_OFFSET_NED.
+      MAV_FRAME_BODY_FLU = 13; // Body fixed frame of reference, Z-up (x: forward, y: left, z: up).
+      MAV_FRAME_LOCAL_FRD = 20; // Forward, Right, Down coordinate frame. This is a local frame with Z-down and arbitrary F/R alignment (i.e. not aligned with NED/earth frame). Replacement for MAV_FRAME_MOCAP_NED, MAV_FRAME_VISION_NED, MAV_FRAME_ESTIM_NED.
+      MAV_FRAME_LOCAL_FLU = 21; // Forward, Left, Up coordinate frame. This is a local frame with Z-up and arbitrary F/L alignment (i.e. not aligned with ENU/earth frame). Replacement for MAV_FRAME_MOCAP_ENU, MAV_FRAME_VISION_ENU, MAV_FRAME_ESTIM_ENU.
+    }
+
+    // Enumeration of estimator types
+    enum MavEstimatorType {
+        MAV_ESTIMATOR_TYPE_UNKNOWN = 0; // Unknown type of the estimator.
+        MAV_ESTIMATOR_TYPE_NAIVE = 1; // This is a naive estimator without any real covariance feedback.
+        MAV_ESTIMATOR_TYPE_VISION = 2; // Computer vision based estimate. Might be up to scale.
+        MAV_ESTIMATOR_TYPE_VIO = 3; // Visual-inertial estimate.
+        MAV_ESTIMATOR_TYPE_GPS = 4; // Plain GPS estimate.
+        MAV_ESTIMATOR_TYPE_GPS_INS = 5; // Estimator integrating GPS and inertial sensing.
+        MAV_ESTIMATOR_TYPE_MOCAP = 6; // Estimate from external motion capturing system.
+        MAV_ESTIMATOR_TYPE_LIDAR = 7; // Estimator based on lidar sensor input.
+        MAV_ESTIMATOR_TYPE_AUTOPILOT = 8; // Estimator on autopilot.
+    }
+
+    uint64 time_usec = 1; // Timestamp (0 to use Backend timestamp).
+    MavFrame frame_id = 2; // Coordinate frame of reference for the pose data.
+    MavFrame child_frame_id = 3;  // Coordinate frame of reference for the velocity in free space (twist) data.
+    float x = 4; // X position.
+    float y = 5; // Y position.
+    float z = 6; // Z position.
+    repeated float q = 7; // Quaternion components, w, x, y, z (1 0 0 0 is the null-rotation).
+    float vx = 8; // X linear speed (m/s).
+    float vy = 9; // Y linear speed (m/s).
+    float vz = 10; // Z linear speed (m/s).
+    float rollspeed = 11; // Roll angular speed (rad/s).
+    float pitchspeed = 12; // Pitch angular speed (rad/s).
+    float yawspeed = 13; // Yaw angular speed (rad/s).
+    repeated float pose_covariance = 14; // Row-major representation of a 6x6 pose cross-covariance matrix upper right triangle. Set first to NaN if unknown.
```

Comment:
Are there protobuf types that we could re-use here?

Reply:
We can use `mavsdk.rpc.telemetry.Quaternion` and `mavsdk.rpc.telemetry.AngularVelocityBody` for example. But I think that in this case we need to place out reused types in one common proto file. Because of it will little bit confusing if user should import, for example `telemetry` to use `mocap`. And also in protos Quaternion used in Camera and Telemetry both now. 
May be need to do some replacing common types in future...
At this time I will try to place some data types in `mocap.proto`. May be it will be ok for now.

Reply:
Ok, I think for now we duplicate the types as you have it here. It's easier for now.

Reply:
`Quaternion`, `Position` and `Covariance` reused now